examples/gallery/src/hexagon-layer.html

```diff
@@ -100,7 +100,7 @@
       OPTIONS.forEach(key => {
         const value = document.getElementById(key).value;
         document.getElementById(key + '-value').innerHTML = value;
-        options[key] = value;
+        options[key] = Number(value);
```

Comment:
@1chandu `HTMLInputElement.value` is a string. luma.gl no longer recasts the type of the uniform.